packages/sdk/src/index.ts

```diff
@@ -1,11 +1,11 @@
 import createClient from "openapi-fetch";
 
-import type { components, paths } from "./hoarder-api.d.ts";
+import type { components, paths } from "./karakeep-api.js";
 
 /**
  * @deprecated Use createKarakeepClient instead.
  */
-export const createHoarderClient = createClient<paths>;
+export const createKarakeepClient = createClient<paths>;
```

Comment:
This should remain as is for backward compatibility. `createKarakeepClient` is already defined underneath it and this API is already marked as deprecated.